packages/integrations/mdx/test/mdx-get-headings.test.js

```diff
@@ -0,0 +1,56 @@
+import mdx from '@astrojs/mdx';
+
+import { expect } from 'chai';
+import { parseHTML } from 'linkedom';
+import { loadFixture } from '../../../astro/test/test-utils.js';
+
+describe('MDX getHeadings', () => {
+	let fixture;
+
+	before(async () => {
+		fixture = await loadFixture({
```

Comment:
Would you mind adding a suite to test removing the default plugin as well?

Would be great to know CI will catch if we accidentally break that code path later assuming the headings are always added

Reply:
Added `rehype-plugins` fixture to mirror our `remark-plugins` fixture!

https://github.com/withastro/astro/pull/4095/files#diff-a70f2d7f157293c364e94828b2a35b9eb4e14dc316330ce5ffe0546613869cad